ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -77,6 +77,13 @@
       "Minimum time between consecutive health check evaluations. Health check queries before "
           + "the interval has elapsed will receive cached responses.";
 
+  static final String KSQL_COMMAND_RUNNER_HEALTH_CHECK_MS =
+          KSQL_CONFIG_PREFIX + "server.command.runner.healthcheck.ms";
```

Comment:
This implies some sort of healtchecking interval. I'd name this something like: `server.command.blocked.threshold.error`

Reply:
Changed it to `server.command.blocked.threshold.error.ms`